### PR TITLE
Mobile 3998

### PR DESCRIPTION
**Pull request by dpalou**

### CODE REVIEW

File: src/addons/mod/imscp/components/index/addon-mod-imscp-index.html [hunk 1]
```diff
@@ -13,25 +9,36 @@
...
+                <ion-label [class.core-bold]="!item.href">
+                    <p>
+                        <span class="ion-padding-start" *ngFor="let i of getNumberForPadding(item.level)"></span>
+                        {{item.title}}
+                    </p>
+                </ion-label>
+            </ion-item>
+        </ion-list>
+
+        <ion-button class="ion-margin" expand="block" (click)="openImscp()">
```
Comment (crazyserver) [inline src/addons/mod/imscp/components/index/addon-mod-imscp-index.html:34]: Recently MOBILE-3149 has been integrated, can you add the buttons to the collapsible-footer here?

**Review by crazyserver — CHANGES_REQUESTED** [review]
Additionally to previous comment: Probably we should open an issue to erase last viewed data from course/activity from the downloads manager.

File: src/addons/mod/imscp/components/index/addon-mod-imscp-index.html (hunk 1 above)
Comment (dpalou) [reply src/addons/mod/imscp/components/index/addon-mod-imscp-index.html:34]: Done.

**Comment by dpalou** [comment]
The last viewed item isn't downloaded data, it could be considered "session" data like the selected order in course overview, so I think it shouldn't be deleted.

**Review by crazyserver — APPROVED** [review]

**Pull request merged**